src/main.rs

```diff
@@ -60,17 +60,14 @@ fn main() {
   // duplicates. We will report duplicates later.
   let mut tags_map = HashMap::new();
   let _ = walk::walk(dir, |path, contents| {
-    // We know that the regex is well-formed, so we are justified in
-    // unwrapping.
```

Comment:
(This comment shouldn't be here. It was left over from some refactoring I did earlier.)